jupyter_client/kernelspecapp.py

```diff
@@ -138,6 +144,61 @@ def start(self):
                 self.exit(1)
             raise
 
+class RemoveKernelSpec(JupyterApp):
+    version = __version__
+    description = """Remove one or more Jupyter kernelspecs by name."""
+    examples = """jupyter kernelspec remove python2 [my_kernel]"""
```

Comment:
As is the convention with command-line args, `[arg]` indicates that it is optional, so this means it can remove several kernelspecs.